.github/CODEOWNERS

```diff
@@ -38,7 +38,7 @@
 # Platform Ops
 /monitoring/opencensus                 @yuriatgoogle @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 /monitoring/prometheus                 @yuriatgoogle @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
-/trace/**/*                            @ymotongpoo @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
+/trace/**/*                            @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
```

Comment:
would be great to have some dedicated team for this

Reply:
Would `googleapi-dataplex`be a good owner?

Reply:
Please check resolved comment in this PR for reference

Reply:
OK, that's Fine :) sorry, didn't read previous comments